bundles/model/org.eclipse.smarthome.model.persistence/OSGI-INF/persistenceextension.xml

```diff
@@ -16,4 +16,5 @@
 <scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" immediate="true" name="org.eclipse.smarthome.model.persistence.extensions">
    <implementation class="org.eclipse.smarthome.model.persistence.extensions.PersistenceExtensions"/>
    <reference bind="setPersistenceServiceRegistry" cardinality="1..1" interface="org.eclipse.smarthome.core.persistence.PersistenceServiceRegistry" name="PersistenceServiceRegistry" policy="static" unbind="unsetPersistenceServiceRegistry"/>
+   <reference bind="setTimeZoneProvider" cardinality="1..1" interface="org.eclipse.smarthome.core.i18n.TimeZoneProvider" name="TimeZoneProvider" policy="static" unbind="unsetTimeZoneProvider"/>
```

Comment:
Since we have switched to DS annotations since a while, could you please convert this xml file to annotations since you are touching it anyway. And while you are at it, would you please be so kind and also convert the `persistencemanager.xml` next to it too? Because then you can place a `.gitignore` in the `OSGI-INF` folder with `/*.xml`.